wordle/terminal_manager.go

```diff
@@ -107,77 +107,66 @@ func NewTerminalManager(ctx context.Context, game *WordGame) *TerminalManager {
 	}
 }
 
-func (ui *TerminalManager) Run() error {
+func (ui *TerminalManager) Run(initialized *bool) error {
 	go ui.handleEvents()
 	defer ui.end()
 
+	go func() {
+		// mark as initialized after ~200 milliseconds
+		time.Sleep(200 * time.Millisecond)
+		*initialized = true
+	}()
```

Comment:
hmm, basing initialization on a timer is not a good idea in my opinion. You should have someplace in your code where you are sure everything is initialized.